MSON Specification.md

```diff
@@ -794,6 +795,32 @@ by using an `optional` attribute and/or MAY indicate values are samples using a
     Implies an `array` type structure that MUST contain "red" as an item and MAY contain any other strings, where
     "green" is a sample value.
 
+With a `fixed-type` _[Type Attribute][]_:
+
+- If a _[Named Type][]_ or _[Member Type][]_ annotates its type as `fixed-type`, _[Nested Member Types][]_ do not inherit
+the `fixed-type` attribute.
+
+- An `array` based _[Named Type][]_ or _[Member Type][]_ MAY specify `fixed-type` to indicate the structure
+MUST contain items of the specified types only.
+
+    ```
+    - colors (array, fixed-type)
+        - (string)
+    ```
+
+    Implies a fixed-list `array` structure that MUST only contain string items.
+
+- An `object` based _[Named Type][]_ or _[Member Type][]_ MAY specify `fixed-type` to indicate an object where all
+the properties MUST be present. Further, such an `object` type structure MUST NOT contain any other properties.
+
+    ```
+    - person (object, fixed-type)
```

Comment:
What is about such examples:

for array
```
- colors (array, fixed-type)
    - red (string)
```
Implies a fixed-list `array` structure that MUST only contain any string items; `red` is an example value.

for object
```
- person (object, fixed-type)
    - first_name: John
    - last_name: Smith
```

Implies an `object` that MUST contain only the properties "first_name" and "last_name", respectively; `John` and `Smith` are example values.